liip/theme-bundle/1.5/config/packages/framework.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+    templating:
+        engines: ['twig']
```

Comment:
ok .. that doesn’t seem to do the trick :-/

Reply:
would it work if I fold it into the theme.yaml?

Reply:
Using the Symfony Templating layer as done here is "forbidden" in Symfony 3.4/4.0. Supporting Twig directly is what is epxected in recipes.

Reply:
ok .. which requires a major refactoring of this Bundle .. which I am currently not planning to work on (see https://github.com/liip/LiipThemeBundle/issues/181).

so I guess I will close this PR